qt/aqt/editor.py

```diff
@@ -1177,6 +1199,40 @@ def toggleCloseHTMLTags(self) -> None:
     def setTagsCollapsed(self, collapsed: bool) -> None:
         aqt.mw.pm.set_tags_collapsed(self.editorMode, collapsed)
 
+    def onAddImageForOcclusion(self) -> None:
+        """Show a file selection screen, then get selected image path."""
+        extension_filter = " ".join(
+            f"*.{extension}" for extension in sorted(itertools.chain(pics))
+        )
+        filter = f"{tr.editing_media()} ({extension_filter})"
+
+        def accept(file: str) -> None:
+            try:
+                html = self._addMedia(file)
+                options = {"kind": "add", "imagePath": file, "notetypeId": 0}
+                # pass both html and options
+                options = {"html": html, "mode": options}
+                self.web.eval(f"setupMaskEditor({options})")
```

Comment:
It would probably be safer to use `json.dumps()` here instead of stock Python format string serialization